### ISSUE TITLE
Unable to run make deps

### ISSUE THREAD

**Issue by hecko** [TheThingsNetwork/lorawan-devices#133]
Hi, Im unable to run the 

```
make deps
```

This is on MacOS 11.2.1 / Big Sur - the output is:

```
Marcels-MacBook-Pro-511:lorawan-devices marcel$ make deps
npm install

up to date, audited 70 packages in 1s

found 0 vulnerabilities
pushd ./tools/runscript && go install . && popd
~/git/lorawan-devices/tools/runscript ~/git/lorawan-devices
main.go:25:2: cannot find package "go.thethings.network/lorawan-stack/v3/pkg/errors" in any of:
	/usr/local/go/src/go.thethings.network/lorawan-stack/v3/pkg/errors (from $GOROOT)
	/Users/marcel/go/src/go.thethings.network/lorawan-stack/v3/pkg/errors (from $GOPATH)
main.go:26:2: cannot find package "go.thethings.network/lorawan-stack/v3/pkg/scripting" in any of:
	/usr/local/go/src/go.thethings.network/lorawan-stack/v3/pkg/scripting (from $GOROOT)
	/Users/marcel/go/src/go.thethings.network/lorawan-stack/v3/pkg/scripting (from $GOPATH)
main.go:27:2: cannot find package "go.thethings.network/lorawan-stack/v3/pkg/scripting/javascript" in any of:
	/usr/local/go/src/go.thethings.network/lorawan-stack/v3/pkg/scripting/javascript (from $GOROOT)
	/Users/marcel/go/src/go.thethings.network/lorawan-stack/v3/pkg/scripting/javascript (from $GOPATH)
make: *** [deps] Error 1
```

I believe this is something that is fixable by any Go programmer, but the Go ecosystem is not too obvious for me unfortunately. 

```
Marcels-MacBook-Pro-511:lorawan-devices marcel$ node -v
v15.14.0
Marcels-MacBook-Pro-511:lorawan-devices marcel$ npm -v
7.9.0
Marcels-MacBook-Pro-511:lorawan-devices marcel$ go version
go version go1.9 darwin/amd64
Marcels-MacBook-Pro-511:lorawan-devices marcel$ go env GOPATH
/Users/marcel/go
```

Thanks for any help.

**Comment by hecko** [comment]
D'oh! Upgrading Go to 1.16 - Ill update this issue in a moment.

**Comment by hecko** [comment]
Please excuse my ignorance and a mistake in running proper Go version - Please close this issue. The fix is obviously to upgrade my 1.9 to >1.15

Marcel

**Comment by hecko** [comment]
Closig issue
